models/public/license-plate-recognition-barrier-0007/license-plate-recognition-barrier-0007.md

```diff
@@ -0,0 +1,219 @@
+# license-plate-recognition-barrier-0007
+
+## Use Case and High-Level Description
+
+This model uses a small-footprint network trained end-to-end to recognize
+Chinese license plates in traffic.
+
+## Validation Dataset - Internal
+
+1165 Chinese plates from different provinces
+
+## Example
+
+**Note:** The license plates on the image were modified to protect the owners' privacy.
+
+![](./license-plate-recognition-barrier-0007.png)
+
+## Specification
+
+| Metric                 | Value                   |
+|------------------------|-------------------------|
+| Rotation in-plane      | ±10˚                    |
+| Rotation out-of-plane  | Yaw: ±45˚ / Pitch: ±45˚ |
+| Min plate width        | 94 pixels               |
+| Ratio of correct reads | 90.12%                  |
+| GFlops                 | 0.347                   |
+| MParams                | 1.435                   |
+| Source framework       | TensorFlow*             |
+
+## Limitations
+
+Only "blue" license plates, which are common in public, were tested
+thoroughly. Other types of license plates may underperform.
+
+## Performance
+Link to [performance table](https://software.intel.com/en-us/openvino-toolkit/benchmarks)
+
+## Inputs
+
+### Original Model
+
+Image,	name: `data` , shape: `1,3,24,94`, format is `1,C,H,W` where:
+
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+### Converted Model
+
+Image,	name: `data` , shape: `1,3,24,94`, format is `1,C,H,W` where:
+
+- `C` - channel
+- `H` - height
+- `W` - width
+
+Channel order is `BGR`.
+
+## Outputs
+
+### Original Model
+
+Encoded vector of floats, name: `decode`, shape: `1,88,1,1`. Each float
+is an integer number encoding a character according to this dictionary:
+
+        0 0
```

Comment:
May be you should move this dictionary above, to **Outputs** section, because of it duplication. Or even to separate file. 
@IRDonch what do you think about it?

Reply:
@eizamaliev I'm on vacation this week, so I won't do it. If you think it's reasonable, please, do it yourself.

Reply:
I think a separate file would be overkill, but pulling it up into **Outputs** seems reasonable.